cms/templates/js/mock/mock-paged-container-xblock.underscore

```diff
@@ -0,0 +1,261 @@
+<header class="xblock-header">
+    <div class="xblock-header-primary">
+        <div class="header-details">
+            <span class="xblock-display-name">Test Container</span>
+        </div>
+        <div class="header-actions">
+            <ul class="actions-list">
+            </ul>
+        </div>
+    </div>
+</header>
+<article class="xblock-render">
+    <div class="xblock" data-locator="locator-container" data-request-token="page-render-token"
+         data-init="MockXBlock" data-runtime-class="StudioRuntime" data-runtime-version="1">
+
+         <script type="text/template" id="paging-header-tpl">
+		    <div class="meta-wrap">
+		    <div class="meta">
+		        <%= messageHtml %>
+		    </div>
+		    <nav class="pagination pagination-compact top">
+		        <ol>
+		            <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label">Previous</span></a></li>
+		            <li class="nav-item next"><a class="nav-link next-page-link" href="#"><span class="nav-label">Next</span> <i class="icon fa fa-angle-right"></i></a></li>
+		        </ol>
+		    </nav>
+		</div>
+
+		</script>
+		<script type="text/template" id="paging-footer-tpl">
+		    <nav class="pagination pagination-full bottom">
+                <ol>
+                    <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label">Previous</span></a></li>
+                    <li class="nav-item page">
+                        <div class="pagination-form">
+                            <label class="page-number-label" for="page-number">Page number</label>
+                            <input id="page-number-input" class="page-number-input" name="page-number" type="text" size="4" />
+                        </div>
+
+                        <span class="current-page"><%= current_page + 1 %></span>
+                        <span class="page-divider">/</span>
+                        <span class="total-pages"><%= total_pages %></span>
+                    </li>
+                    <li class="nav-item next"><a class="nav-link next-page-link" href="#"><span class="nav-label">Next</span> <i class="icon fa fa-angle-right"></i></a></li>
+                </ol>
+		    </nav>
+		</script>
+
+		<div class="container-paging-header"></div>
+
+        <div class="studio-xblock-wrapper" data-locator="locator-group-A">
+            <section class="wrapper-xblock level-nesting">
+                <header class="xblock-header">
+                    <div class="xblock-header-primary">
+                        <div class="header-details">
+                            <a href="#" data-tooltip="Expand or Collapse" class="action expand-collapse expand">
+                                <i class="icon fa fa-caret-down ui-toggle-expansion"></i>
+                                <span class="sr">Expand or Collapse</span>
```

Comment:
This value needs to be toggled ("Expand" if the region is collapsed, and "Collapse" if the region is expanded).